src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,144 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { useEffect, useMemo, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(false);
```

Comment:
if you have a boolean value then better to start the name with `is` or `has`, like `isLoading`